src/pages/timecard/Timecard.jsx

```diff
@@ -20,11 +20,26 @@ const Timecard = () => {
     document.title = generateDocumentTitle('Timecard');
   });
 
+  const sortSummaryErrors = () => {
```

Comment:
Given it's been identified as a bug that it's possible to get out of order errors, should we have some sort of test to verify that we're always getting sorted errors.

Reply:
I've now added some tests to cover the new functionality